src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.subsystems.hood;
+
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class HoodConstants {
+    public static Translation2d mechanism2dPose = new Translation2d(1, 1);
+    public static double HoodLength = 0.4; // m
+    public static double velocityTolerance = 0.5;
```

Comment:
Not used anywhere

Reply:
d8fb3a02238c3de747945b43b22cd8baffd009fb

Reply:
But why? This doesn't have any advantage.

Reply:
There should be some way to understand the units of a measure. If it's dimensionless you can use `Measure<Dimensionless>` or write a comment (but then you need to jump up the constants file to see the unit).
Also all constants need to be all caps

Reply:
56f6627f614465fc556a251028345bb5a5207df5

Reply:
whoops

Reply:
edeee7f63861dfedc89f2fcaab2e395d9a1e555b

Reply:
How can you convert dimensionless to volts...?

Reply:
Also if you convert a constant just save the converted value in the constant.

Reply:
Volts?

Reply:
https://github.com/Galaxia5987/robot-2024/blob/edeee7f63861dfedc89f2fcaab2e395d9a1e555b/src/main/java/frc/robot/subsystems/hood/HoodConstants.java#L11
https://github.com/Galaxia5987/robot-2024/blob/edeee7f63861dfedc89f2fcaab2e395d9a1e555b/src/main/java/frc/robot/subsystems/hood/Hood.java#L54

Reply:
Ohhh it's "value". Sorry. read "volts".